otter/test/convergence/test_service.py

```diff
@@ -817,7 +823,8 @@ def setUp(self):
                    links=freeze([{'href': 'link2', 'rel': 'self'}]))
         )
         self.state_active = {}
-        self.cache = [thaw(self.servers[0].json), thaw(self.servers[1].json)]
+        self.cache = [thaw(self.servers[0].json.set('_is_as_active', True)),
+                      thaw(self.servers[1].json.set('_is_as_active', True))]
```

Comment:
I may be failing to understand but based on above `update_cache` changes I don't think servers need to become AS active.